diff --git a/src/util/inquire.js b/src/util/inquire.js
index 5b97063e0ad4014b6aeb81ece93f59c025763fea..e764a7f90b29edaf5d823c0d20596433757772d4 100644
--- a/src/util/inquire.js
+++ b/src/util/inquire.js
@@ -13,7 +13,7 @@ function inquire(moduleName) {
     if (typeof require !== "function") {
       return null;
     }
-    var mod = require(moduleName);
+    var mod = eval("require")(moduleName); // eslint-disable-line no-eval
     if (mod && (mod.length || Object.keys(mod).length)) return mod;
     return null;
   } catch (err) {
